### DISCUSSION TITLE
Dont update (west update) (kit nRF9160)

### DISCUSSION THREAD

**Discussion by sennet-info** [1NCE-GmbH/blueprint-zephyr#9] [Q&A]
Hi, i'm starting with development with a mRF9160 kit. I'm try to follow all steps, but something was wrong. Please help me.

![issue_point](https://github.com/user-attachments/assets/d380b853-f030-4136-9e5e-5716d15b6c8a)
![issue_point_report](https://github.com/user-attachments/assets/2fafc7d8-74c6-4c69-b223-6453ea219558)

dont' progress with "west update"

I have change two files as you details:

![files_changes](https://github.com/user-attachments/assets/73d1b7b3-59dc-4456-ae78-0edf6cee3d2d)

how can I progress?

thanks in advance.

**Comment by jamali005** [comment]
Hi @sennet-info, 
Glad to assist you with your issue you’re experiencing with the "west update" command. Here are a few suggestions to address the problem:

**1. Copy Lines Individually:** Ensure you copy the lines to example.yaml one by one. This can help avoid any hidden formatting issues that may arise if you copy the entire block at once.

**2. Version Compatibility:** It appears you are using an updated version of nRF Connect. Please note that our release was tested on version 2.2.0, so the updated version could be causing some compatibility issues.

If you continue to face problems, please don’t hesitate to reply to this thread.

**Comment by sennet-info** [comment]
Thanks for your answer, when I be able to install v2.2.0 nRF Connect, but when I try to install Toolchain that's a requeriment, I can't.

![toolchain-error](https://github.com/user-attachments/assets/42b03dad-e0b8-4287-8e47-c0e8942aeef8)

Any idea.

**Comment by jamali005** [comment]
Hi @sennet-info 

It seems there was a misunderstanding. I meant that you should install the nRF Connect SDK v2.2.0 using the nRF Toolchain Manager. Here are the steps to do so:

1. Open the nRF Connect for Desktop application.
2. Go to the Toolchain Manager.
3. Select the nRF Connect SDK version 2.2.0 from the list.
4. Proceed with the installation.

**Comment by sennet-info** [comment]
Ok thanks for your support.
Right now in my personal computer I get to run all APP's (UDP/COAP/LWM2M), I've send to 1nce throught 
UDP.
But in the computer of my office I get an error when I try to do the same steps.
![imagen](https://github.com/user-attachments/assets/8937e031-954d-47d4-86f6-fe7e04f06405)

Could help me with this?

I need to use the computer of my office, but I'm not sure what's wrong.

Thanks in advance.

**Reply by jamali005** [reply]
Hi @sennet-info,

Apologies for the delay in responding. I hope the issue has been resolved on your end. If not, please note that we will soon be releasing compatibility with the new version.

Thank you for your patience and understanding.
